docs/plugins/woodpecker-plugins/plugins.json

```diff
@@ -60,4 +65,4 @@
       "verified": true
     }
   ]
-}
+}
```

Comment:
```suggestion
}

```

Reply:
please respect the .editorconfig !

looks like your IDE/editor does not -> https://editorconfig.org/

Reply:
https://ci.woodpecker-ci.org/woodpecker-ci/woodpecker/build/3971/23